src/lib/stack-client.js

```diff
@@ -1,3 +1,29 @@
 import internal from 'lib/stack-internal.js'
 
-export default internal
\ No newline at end of file
+/**
+ * Cozy client instance
+ * @private
+ */
+let cozyClient
+
+/**
+ * Initializes the functions to call the cozy stack
+ *
+ * @function
+ * @param {Object}  arg
+ * @param {Object}  arg.cozyClient - a cozy client instance
+ * @param {Function} arg.onCreateApp 
+ * @param {Function} arg.onDeleteApp
+ * @returns {Promise}
+ */
+const init = function(options) {
+  cozyClient = options.cozyClient
+  const legacyOptions = {
```

Comment:
`const stackClient = getStackClient()` so that it is not repeated.

Reply:
The last stage will only have one call to `getStackClient()` so the repetition is only visible between commits of this PR